core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/EntityApiServiceSpec.scala

```diff
@@ -1615,6 +1615,65 @@ class EntityApiServiceSpec extends ApiServiceSpec {
         assertResult(StatusCodes.Conflict) {
           status
         }
+
+        assertResult(EntityCopyResponse(Seq.empty, Seq(EntityHardConflict("Sample", "sample1")), Seq.empty)) {
+          responseAs[EntityCopyResponse]
+        }
+      }
+  }
+
+  it should "return 409 for copying entities into a workspace with subtree conflicts, but successfully copy when asked to" in withTestDataApiServices { services =>
+    val sourceWorkspace = WorkspaceName(testData.workspace.namespace, testData.workspace.name)
+    val entityCopyDefinition1 = EntityCopyDefinition(sourceWorkspace, testData.controlledWorkspace.toWorkspaceName, testData.sample1.entityType, Seq(testData.sample1.name))
```

Comment:
Add another test which copies `aliquot1` into an empty workspace and verify that also copying `sample3` causes a Soft Conflict